tests/test_desktop_asset_patterns.py

```
@@ -161,15 +161,27 @@ def test_should_download_asset_with_excludes(self, mocker):
             "SELECTED_DESKTOP_ASSETS": [],
         }
 
         mock_cache = MagicMock()
         downloader = MeshtasticDesktopDownloader(config, mock_cache)
 
-        # When no patterns selected, all should match
+        assert downloader.should_download_asset("Meshtastic-2.7.14.dmg") is False
+        assert downloader.should_download_asset("Meshtastic-2.7.14.msi") is False
+        assert downloader.should_download_asset("Meshtastic-2.7.14.exe") is False
+
+    def test_should_download_asset_wildcard(self, mocker):
```

Ruff (0.15.12)

[warning] Unused method argument: `mocker`

(ARG002)